test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -227,6 +230,38 @@ localContTest iot nio = testCaseSteps "local continuation test" $ \step -> do
         $ mkCont
         $ mkContMsg pid 1
 
+pollingConfirmDepth :: IO (Time Micros) -> IO ChainwebNetwork -> TestTree
+pollingConfirmDepth iot nio = testCaseSteps "poll confirmation depth test" $ \step -> do
+    cenv <- _getServiceClientEnv <$> nio
+
+    step "execute /send with initial pact continuation tx"
+    cmd1 <- firstStep
+    rks <- sending sid cenv (SubmitBatch $ pure cmd1)
+
+    step "check /poll responses to extract pact id for continuation"
+
+    beforePolling <- getCurrentBlockHeight v cenv sid
+    PollResponses _ <- pollingWithDepth sid cenv rks (Just $ ConfirmationDepth 70) ExpectPactResult
+    afterPolling <- getCurrentBlockHeight v cenv sid
+
+    case (beforePolling, afterPolling) of
+      (Just bf, Just af) ->
+        assertBool "the difference between heights should be no less than the confirmation depth" $ (af - bf) >= 70
+      _ -> assertFailure "couldn't get block heights"
+  where
+    sid = unsafeChainId 0
+    tx =
+      "(namespace 'free)(module m G (defcap G () true) (defpact p () (step (yield { \"a\" : (+ 1 1) })) (step (resume { \"a\" := a } a))))(free.m.p)"
```

Comment:
We can simplify this a bit from `localContTest` above. We don't actually care what's in `tx`, as long as it completes. It can even fail. I recommend something simple like `tx = "42"`, to make it clearer that it doesn't matter what it does.

Reply:
Agree, thanks!